content/blog/dplyr-count/index.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+output: hugodown::hugo_document
+
+slug: dplyr-count
+title: dplyr 1.0.1 and the `count()` function
+date: 2020-07-27
+author: Hadley Wickham
+description: >
+    dplyr 1.0.1 is out now — the main change is that `dplyr::count()` now
```

Comment:
Just to confirm, we're going "newspaper" style and using spaces around em dashes?

Reply:
Hmmm I guess that's the style I use when it's non-parentical

Reply:
I like it as well. Asymmetric em-dash spacing, however, is a capital offense.